README.md

```diff
@@ -50,7 +56,6 @@ You can also see the status of the bootkube.sh script which is running via
 
 ## 2) Cleanup
 
-To clean up your environment you can run:
+To clean up your environment, run::
 
-- `./ocp_cleanup.sh`
-- `./libvirt_cleanup.sh`
+make clean
```

Comment:
I'm fine with adding the make clean reference, but can we please leave the reference to the underlying scripts as well - most developers won't want to run make clean every time, and instead will iterate on something like:

```
./ocp_cleanup.sh
rm -fr ocp
./05_run_ocp.sh
```

So lets say something like 

- To clean up the ocp deployment run `./ocp_cleanup.sh`
- To clean up the dummy baremetal VMs and associated libvirt resources run `./libvirt_cleanup.sh`

Or, you can run `make clean` which will run all of the cleanup steps.

Reply:
Hi! I'm still testing this and adding a dev-run option with the 3 steps will come next :)